website/content/en/preview/concepts/nodeclasses.md

```diff
@@ -639,6 +650,9 @@ The disks must be formatted & mounted in a RAID0 and be the underlying filesyste
 
 On AL2, Karpenter automatically configures the disks through an additional boostrap argument (`--local-disks raid0`). The device name is `/dev/md/0` and its mount point is `/mnt/k8s-disks/0`. You should ensure any additional disk setup does not interfere with these.	
 
+### AL2023
+On AL2023, local disks are configured as raid0 by default, regardless of this field's value. Like AL2, the device name is `dev/md/0` and its mount point is `/mnt/k8s-disks/0`. You should ensure any additional disk setup does not interfere with these.
```

Comment:
If this is the case, this changes our ephemeral storage calculation that we send back from the GetInstanceTypes() call. We should change that. Additionally, if this is the case, is this disable-able? If it's not, is it sensical to be able to set the instanceStorePolicy here?

Reply:
I don't believe it's configurable but I'll follow up with the AMI folks to verify. If it isn't it wouldn't make sense to set one way or another, that's why I mentioned that it is raid0 by default regardless of this fields value. Maybe that could be more emphasized.

Reply:
Were you asking if we should enforce that a user can't set the instanceStorePolicy when using AL2023? I still have to check and ensure that this is not configurable but otherwise I think it would be reasonable to enforce this through a CEL validation. The only counter point I think I have is that a user could theoretically do something via custom user data even if the knob is not explicitly surfaced through the AMI.

Reply:
Resolved, instance store policy is now surfaced by the AL2023 AMI, behavior will now match AL2.